app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file, new_file = command.split()
+    if "mv" in command:
+        directory = os.path.split(new_file)
+        parent = ""
+        if len(directory) > 1:
+            for path in directory[:-1]:
+                parent = os.path.join(parent, path)
+                os.mkdir(parent)
```

Comment:
Please, use `os.mkdirs` instead of `os.mkdir`. Read more: https://note.nkmk.me/en/python-os-mkdir-makedirs/